Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query_test.py

```diff
@@ -856,3 +856,43 @@ def modify_dn(self, dn, cn, new_superior):
     connection_mocker = mocker.patch.object(Active_Directory_Query.connection, 'modify_dn', return_value=True)
     Active_Directory_Query.modify_user_ou(dn, new_ou)
     assert connection_mocker.call_args[0][1] == expected
+
+
+def test_search_users_with_msDSUserAccountControlComputed(mocker):
+    """
+    Given:
+        The 'msDSUserAccountControlComputed' was returned.
+    When:
+        Run the 'ad-get-user' command
+    Then:
+        The user_account_to_boolean_fields_msDS_user_account_control_computed was called.
+    """
+
+    import Active_Directory_Query
+
+    class EntryMocker:
+        def entry_to_json(self):
+            return (
+                '{"attributes": {"displayName": [], "mail": [], "manager": [], "memberOf": ["memberOf"], '
+                '"name": ["Guest"], "sAMAccountName": ["Guest"], "userAccountControl": [], \
+                   "msDS-User-Account-Control-Computed": [0]},"dn": "test_dn"}'
+            )
+
+    class ConnectionMocker:
+        entries = [EntryMocker()]
+        result = {"controls": {"": {"value": {"cookie": b"<cookie>"}}}}
+
+        def search(self, *args, **kwargs):
+            time.sleep(1)
+
+    mocker.patch.object(demisto, "args", return_value={"page-size": "1"})
+    mocker_msDSUserAccountControlComputed = mocker.patch.object(
+        Active_Directory_Query,
+        "user_account_to_boolean_fields_msDS_user_account_control_computed",
+        return_value={},
+    )
+
+    Active_Directory_Query.connection = ConnectionMocker()
+
+    Active_Directory_Query.search_users("dc", 1)
+    mocker_msDSUserAccountControlComputed.assert_called_once()
```

Comment:
Lets check also that the correct value was returned, not only that the method was called.